components/director/pkg/graphql/schema.graphql

```diff
@@ -1204,6 +1204,7 @@ type Tenant {
 	parentID: ID
 	initialized: Boolean
 	labels(key: String): Labels
+	provider: String!
```

Comment:
Currently, in the hydrator component, the new object context provider will check if the tenant has a subdomain label and if it does not exist will update it through `writeTenants` mutation, which will update the tenant with whatever we provide as parameters. So we need to be sure the `getTenants` query returns everything that `writeTenants` accepts as parameters to avoid overriding the tenant fields with wrong information.